src/db/models/Commit.ts

```diff
@@ -8,6 +8,8 @@ export interface Commit {
   score: number;
   message: string;
   committed_at: string;
+  repo_name: string;
+  ref: string;
```

Comment:
What is the meaning/usage of this field?

Reply:
For the git ref that the Commit was pushed on

Reply:
I agree that repository name is needed, but I can't think of a case where the git ref (branch/tag/etc.) is needed for now. Do you have a scenario in mind?

Reply:
Might be useful if we only want to look at commits from a certain branch (i.e. master) but think this is an unlikely scenario. I can remove this column

Reply:
We can remove it for now then.